system-parachains/asset-hubs/asset-hub-polkadot/Cargo.toml

```diff
@@ -121,7 +121,6 @@ parachain-info = { workspace = true }
 parachains-common = { workspace = true }
 system-parachains-common = { workspace = true }
 assets-common = { workspace = true }
-polkadot-runtime = { workspace = true }                                   # Pulling this in makes it easier to reference Proxy type etc
```

Comment:
This broke integration tests on the AHM branch:

Since this PR, all polkadot chain runtimes are failing compilation in emulated tests

Reply:
fixed in https://github.com/polkadot-fellows/runtimes/pull/722